include/seqan3/range/view/take.hpp

```diff
@@ -536,6 +463,47 @@ struct take_fn
                 target_size = std::min(target_size, std::ranges::size(urange));
             }
         }
+
+        // string_view
+        if constexpr (is_type_specialisation_of_v<remove_cvref_t<urng_t>, std::basic_string_view>)
+        {
+            return urange.substr(0, target_size);
+        }
+        // string const
+        else if constexpr (is_type_specialisation_of_v<remove_cvref_t<urng_t>, std::basic_string> &&
+                           std::is_const_v<std::remove_reference_t<urng_t>> &&
+                           std::is_lvalue_reference_v<urng_t>)
```

Comment:
is_lvalue_reference does not make sense in this context, does it? The urng_t is a std::basic_string and cannot be a temporary by the viewable range concept? Or what I am missing?

Reply:
It's not required, but I thought it would help understand what's goinf on (that we really only have & types here). But I can remove it.

Reply:
moved this to the documentation line above now

Reply:
Yeah, I am not sure it helps understanding because one immediately asks why this protection. Is there an overload I am not seeing? So I think it is better to remove it.